### ISSUE TITLE
ListObjectsV2 return invalid date format

### ISSUE THREAD

**Issue by arnaudbriche** [NVIDIA/aistore#190]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the bug

Hi,

Try to use aws cli to query my ais deployment.
While trying to do a ls of a bucket, I'm getting the following error:

```
Invalid timestamp ":2024-10-11T13:22:41Z": ('Unknown string format:', ':2024-10-11T13:22:41Z')
```

Indeed, the response body contains weird XML LastModified field values:

```xml
<LastModified>:2024-10-11T13:22:41Z</LastModified>
```

### Expected Behavior

It should not give me an error

### Current Behavior

I gives a parsing error

### Steps To Reproduce

```
aws s3 --endpoint-url=<MY_ENDPOINT ls s3://<BUCKET>/
```

### AIStore build/version

version 3.25.d2feb38cf (build: 2024-10-15T19:31:09+0000)

### Environment details (OS name and version, etc.)

Ubuntu 24.04 / Docker

**Comment by alex-aizman** [comment]
must be either ISO 8601 or RFC 3339...

I'll look into it when time permits

**Comment by arnaudbriche** [comment]
Hi @gaikwadabhishek ,

I just followed the doc about building your own image:

```sh
git clone https://github.com/NVIDIA/aistore.git
git checkout 8b213acac5f64764cbc1d570e46e4d6c8ffa696a
deploy/prod/docker/single
IMAGE_REPO=arnaudbriche/aistorage-cluster-minimal IMAGE_TAG=v3.26-8b213ac  make -e all
```

**Comment by rkoo19** [comment]
Hey @arnaudbriche,

The issue was caused by a minor bug in the date formatting related to parsing RFC3339 timestamps. This has been fixed in [commit cc010a19](https://github.com/NVIDIA/aistore/commit/cc010a19cb9db4a8bfa2581dcfa64bae3761299c). Please pull the latest changes from upstream to resolve the error.

I will be closing this issue for now, but please reach out if you need any further assistance.

**Comment by rkoo19** [comment]
Hey @arnaudbriche,

The issue was due to an index error in how we were parsing the LastModified field from custom metadata (LastModified:2024-10-01T13:23:20Z). This has been addressed in [commit cc010a19](https://github.com/NVIDIA/aistore/commit/cc010a19cb9db4a8bfa2581dcfa64bae3761299c). Please pull the latest changes from upstream to resolve the error.

I will be closing this issue for now, but please reach out if you need any further assistance.
